packages/edit-site/src/components/page-pages/index.js

```diff
@@ -26,13 +26,23 @@ const EMPTY_OBJECT = {};
 export default function PagePages() {
 	const [ view, setView ] = useState( {
 		type: 'list',
-		search: '',
+		filters: {
+			search: '',
+			status: 'publish, draft',
+		},
 		page: 1,
 		perPage: 5,
 		sort: {
 			field: 'date',
 			direction: 'desc',
 		},
+		layout: {
+			filters: [
+				{ type: 'search', id: 'search' },
+				{ type: 'in', id: 'author' },
+				{ type: 'in', id: 'status' },
```

Comment:
I think for this filter the type should be enumeration instead of "in", in makes me think it is filter that forces the value to be in a given set. I expect in the future for an enumeration we will support things. status is draft, status is not trashed status is empty status is not empty. The UI component will probably allow choosing between "is", "is not" etc, in makes me think only on the is case.